cmd/ndt7-client/internal/summary.go

```diff
@@ -0,0 +1,87 @@
+package internal
+
+import (
+	"strings"
+
+	"github.com/m-lab/ndt7-client-go"
+	"github.com/m-lab/ndt7-client-go/spec"
+)
+
+// valueUnitPair represents a {"value": ..., "unit": ...} pair.
+type valueUnitPair struct {
+	Value float64
+	Unit  string
+}
+
+// Summary is a struct containing the values displayed to the user at
+// the end of an ndt7 test.
+type Summary struct {
+	// FQDN of the server used for this test.
+	Server string
+	Client string
+	// Download speed, in Mbit/s. This is measured at the receiver.
+	Download valueUnitPair
```

Comment:
I tend to find it more readable when there is a newline after each field, if possible